### PR TITLE
rpc documentation

### PR DESCRIPTION
**Pull request by karthik1900**

### CODE REVIEW

File: api-reference/remote-run.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+### Example 2 - Run a class on a remote
+    
+```python
+# app.py
+
+import os
+import inferless
+
+InferlessCls = inferless.Cls()
```
Comment (nickaggarwal) [inline api-reference/remote-run.mdx:135]: Why do we have to do this ?

File: api-reference/remote-run.mdx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        return {"generated_result": result_output[0]}
+
+
+model = InferlessPythonModel()
+print(model.infer({"prompt": "Hello, Write a story about a dragon"}))
+```
+
+```yaml
+# config.yaml
+gpu:
```
Comment (nickaggarwal) [inline api-reference/remote-run.mdx:172]: can we take in this in the class for function param, as I don't want to have a file specified in the existing config.yaml

File: api-reference/remote-run.mdx (hunk 1 above)
Comment (karthik1900) [reply api-reference/remote-run.mdx]: modified InferlessCls to `app`

File: api-reference/remote-run.mdx (hunk 2 above)
Comment (karthik1900) [reply api-reference/remote-run.mdx:162]: moved gpu param to the decorator

**Pull request merged**